docs/source/algorithms/MuonMaxent-v1.rst

```diff
@@ -0,0 +1,64 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm calculates a single frequency spectrum from the time domain spectra recorded by multiple groups/detectors. 
+
+The time domian data :math:`D_k(t)`, where :math:`t` is time and :math:`k` is the spectrum number, has associated errors :math:`E_k(t)`. If the number of points chosen is greater than the number of time domain sata points then extra points are
+added with infinite errors. The time domain data prior to :code:`FirstGoodTime` also have their errors set to infinity. The algorithm will produce the frequency spectra :math:`f(\omega)` and this is assumed to be real and positive. 
+The upper limit of the frequency spectra is determined by :code:`MaxField`. The maximum frequency, :math:`\omega_\mathrm{max}` can be less than the Nyquist limit :math:`\frac{\pi}{\delta T}` if the instrumental frequency response function for 
+:math:`\omega>\omega_\mathrm{max}` is approximatley zero. The inital estimate of the frequency spectrum is flat. 
+
+The algorithm calculates an estimate of each time domain spectra, :math:`g_k(t)` by the equation
+
+.. math::  g_k(t)=(1+A_k \Re(\mathrm{IFFT}(f(\omega) R(\omega))\exp(-j\phi_k) ) ),
+
+where :math:`\Re(z)` is the real part of :math:`z`, :math:`\mathrm{IFFT}` is the inverse fast Fourier transform, :math:`\phi_k` is the phase and :math:`A_k` is the asymmetry of the of the  :math:`k^\mathrm{th}` spectrum. 
```

Comment:
There exist two ways of defining IFFT, e.g. from https://www.gnu.org/software/gsl/doc/html/fft.html there is the paragraph: "In general there are two possible choices for the sign of the exponential in the transform/ inverse-transform pair. GSL follows the same convention as FFTPACK, using a negative exponential for the forward transform. The advantage of this convention is that the inverse transform recreates the original function with simple Fourier synthesis. Numerical Recipes uses the opposite convention, a positive exponential in the forward transform.". Which one is used here?

Reply:
I will double check in the morning

Reply:
So looking at the code we dont seem to use the FFT module (I should try to delete it) and instead numpy is used. According to the docs https://docs.scipy.org/doc/numpy-1.12.0/reference/routines.fft.html#module-numpy.fft the forward transform has a minus sign

Reply:
OK. Note I am away Thursday-Friday this week. Simply e.g. adding the above reference to number to numpy's definition I would say is fine